### DISCUSSION TITLE
Moving the position of the knife button on 3ds

### DISCUSSION THREAD

**Discussion by Ellis243454** [nzp-team/nzportable#535] [General]
So on the 3ds recently the knife button was unbounded from the down dpad button to fix a bug but in the process I feel it made it a lot more uncomfortable to knife zombies I feel to make it feel similar again changing the position of the knife button to the left side of the touchscreen in the same position instead of having on the right would make it feel similar again if it's not possible I understand

**Comment by SilentShadows77** [comment]
In general could we get better remapping qnd rebinding options for the 3ds than literally just the quake control/settings menu?
